spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -110,6 +120,27 @@ object DynamicInterpreter {
     optimizerTimeTracker.cleanupWorkaroundDf(conformedDf)
   }
 
+  private def checkMutabilityNotViolated(schema: StructType, steps: List[ConformanceRule])
+                                        (implicit ictx: InterpreterContext): Unit = {
+    val rulesInViolation = findOriginalColumnsModificationRules(steps, schema)
+
+    if (rulesInViolation.nonEmpty) {
+      val violationsString = rulesInViolation.map(rule =>
+        s"Rule number ${rule.order} - ${rule.getClass.getSimpleName}"
+      ).mkString("\n")
+      if (ictx.featureSwitches.allowOriginalColumnsMutability) {
+        log.warn(
```

Comment:
The setting is in reference or app conf. It is not a CLI option. User might not be aware

Reply:
I think it's still OK. Because presumably done by higher authority, UI, when implemented validation, can still warn about that.
The way how it is done, it's not possibly to have rule modifying an original column without a warning or worse. For users (not Absa) who are OK with such behavior, that's a false flag.

Reply:
This is spark-jobs application conf. UI can not warn them about this being on or off. This would take us to a whole another topic of having a REST endpoint with the config in it.

And it is hardly a false flag. It is a warning ([def][log4jwarn]). It is meant to be there if users are doing something dangerous and potentially harmful and something that we do not fully test. When we implement a full suite of Unit or other tests, let's change this.

[log4jwarn]: https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html#WARN

Reply:
I don't want this to be a blocking issue. 😃 
But my point is, if the admin of the application says (configures) it's ok to modify the original columns, I still believe the warning is unjustified. Unless we would have three levels of settings 😉 
To explain my UI remark: exactly because UI wouldn't know about the eventual setting of _SpakJobs_, the validation there (once we have it) should allow such a rule, but warn about potential problem.

Reply:
Other team members? @dk1844 @AdrianOlosutean @HuvarVer ? 

Do you have any thoughts or do you not care in this instance?

Reply:
I would agree to remove the warning now and add it to UI, which can be done in another task